--- PokerTH-0.9.5-src/src/net/common/ircthread.cpp.orig	2012-07-12 22:16:13.656767561 +0200
+++ PokerTH-0.9.5-src/src/net/common/ircthread.cpp	2012-07-12 22:16:37.076766733 +0200
@@ -20,6 +20,7 @@
 #include <net/ircthread.h>
 #include <net/socket_msg.h>
 #include <libircclient/libircclient.h>
+#include <libircclient/libirc_rfcnumeric.h>
 #include <boost/algorithm/string/predicate.hpp>
 #include <queue>
 #include <sstream>
